jbpm-flow/src/main/java/org/jbpm/process/instance/impl/JavaAction.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.process.instance.impl;
+
+import java.io.Serializable;
+
+import org.jbpm.workflow.core.Node;
+
+public interface JavaAction extends Serializable, Action {
+
+    void dumpXML(Node node, StringBuilder xmlDump);
```

Comment:
remove this. An action is not the place to put dumper logic

Reply:
ok, I was trying to force all future writers of a java action to remember they have to provide the dump logic or the engine  will crash at runtime.